tests/test_monitor.py

```diff
@@ -34,3 +35,47 @@ def test_monitor():
     assert set(last_logline.keys()) == {'l', 't', 'r'}, "Incorrect keys in monitor logline"
     file_handler.close()
     os.remove(mon_file)
+
+def test_monitor_load_results(tmp_path):
+    """
+    test load_results on log files produced by the monitor wrapper
+    """
+    env1 = gym.make("CartPole-v1")
+    env1.seed(0)
+    mon_file1 = os.fspath(tmp_path / "stable_baselines-test-{}.monitor.csv".format(uuid.uuid4()))
+    menv1 = Monitor(env1, mon_file1)
+
+    mon_files = get_monitor_files(tmp_path)
+    assert len(mon_files) == 1
+    assert mon_file1 in mon_files
+
+    menv1.reset()
+    for _ in range(1000):
+        _, _, done, _ = menv1.step(0)
```

Comment:
even though it works here, it is usually a better practice to do:
```python
action = env.action_space.sample()
```

Reply:
ok. I will update. I just copied that code from the existing test to not deviate too much.

Reply:
It seems that the test comes from the original baselines and was apparently not updated...